### PR TITLE
Avoid verbose output when we have nothing to say in ProcessFailedException

### PR DESCRIPTION
**Pull request by weitzman**
Fixes the Process part of https://github.com/drush-ops/drush/pull/4642

### Overview
This pull request:

| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | no
| Has tests?    | no
| BC breaks?    | no     
| Deprecations? | no

### CODE REVIEW

File: src/SiteProcess.php [hunk 1]
```diff
@@ -202,6 +204,19 @@ public function start(callable $callback = null, array $env = [])
         parent::start($callback, $env);
     }
 
+    public function mustRun(callable $callback = null, array $env = []): \Symfony\Component\Process\Process
```
Comment (greg-1-anderson) [inline src/SiteProcess.php:207]: Did you mean to have a return typehint here? I believe the main branch is on Symfony 4, pending #50, which is what Drush is using as well, correct? And that has no return typehints?

The other issue we have is that this project is still set up to test on Travis, which is why the tests are not running. 😞

Comment (weitzman) [reply src/SiteProcess.php:207]: I just copied the same method from my symfony/process, which is  v4.4.19 (using Drupal 9, so I can get latest code from this project). Happy to remove the type hint if desired. I'm not seeing any complaints when I do (PHP 7.3).

Comment (weitzman) [reply src/SiteProcess.php:207]: Oops, now I see that you refer to the return hint. That was done by PHPStorm, but I think it is correct. The parent supermethod has `self` as return type hint, so this method also needs one, and  `self` gives the following error

```
Fatal error: Declaration of Consolidation\SiteProcess\SiteProcess::mustRun($callback = NULL, $env = Array): Consolidation\SiteProcess\SiteProcess must be compatible with Symfony\Component\Process\Process::mustRun(?callable $callback = NULL, array $env = Array): Symfony\Component\Process\Process in /Users/moshe.weitzman/drush/vendor/consolidation/site-process/src/SiteProcess.php on line 25
```

**Review by greg-1-anderson — APPROVED** [review]
OK looks good; could you rebase so that we can confirm the tests are passing?

**Pull request merged**